pkg/driftctl.go

```diff
@@ -111,6 +112,8 @@ func (d DriftCTL) Run() (*analyser.Analysis, error) {
 	if !d.opts.StrictMode {
 		middleware = append(middleware,
 			middlewares.NewAwsDefaults(),
+			middlewares.NewAwsDefaultNetworkACL(),
+			middlewares.NewAwsDefaultNetworkACLRule(),
```

Comment:
Don't understand why you would want these two middlewares to be in the strict-mode.

If you want to manage and see diffs on your default network acl you'd need to enable strict-mode and deep-mode.

Why not run both like the others (e.g. default-vpc, default-rout-table ...) ?

Reply:
Yeah you are right I messed up with strict mode scope in my mind thanks 👍🏻